src/components/itemDetailsModal.js

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Button, Modal, ModalContent, Table } from 'semantic-ui-react'
+
+import { ONE_MB } from '../constants'
+import { setActiveItems } from '../actions/itemActions'
+
+import file from './css/details-modal.css'
+
+class ItemDetailsModal extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      active_item: null
+    }
+  }
+
+  componentDidUpdate = prevProps => {
+    const { activeItems } = this.props
+    if (JSON.stringify(prevProps.activeItems) != JSON.stringify(activeItems)) {
+      if (activeItems.length == 1) {
+        this.setState({
+          active_item: activeItems[0]
+        })
+      }
+    }
+  }
+
+  formatDate (string) {
+    var options = { year: 'numeric', month: 'long', day: 'numeric' }
+    return new Date(string).toLocaleDateString([], options)
+  }
+
+  render () {
+    const { showModal, close, filemanager, currentFolder } = this.props
+    console.log(currentFolder)
```

Comment:
remove console statements.
please search for console statements before making pr.